tensorflow/core/kernels/data/experimental/scan_dataset_op.cc

```diff
@@ -106,7 +106,12 @@ class ScanDatasetOp : public UnaryDatasetOpKernel {
 
     string DebugString() const override { return "ScanDatasetOp::Dataset"; }
 
-    int64 Cardinality() const override { return input_->Cardinality(); }
+    int64 Cardinality() const override {
+      if (preserve_cardinality_)
+        return input_->Cardinality();
+      else
```

Comment:
Google C++ style requires using braces when there is an `else` statement: https://google.github.io/styleguide/cppguide.html#Conditionals

Reply:
also applies in map_dataset_op and parallel_map_dataset_op

Reply:
fixed.